packages/viewer/src/lib/interactions/interactionApi.ts

```diff
@@ -404,57 +404,54 @@ export abstract class InteractionApi<T extends Camera = Camera> {
     delta: Point.Point,
     point: Point.Point
   ): Promise<void> {
-    return this.transformCamera(
-      ({ camera, viewport, boundingBox, depthBuffer }) => {
-        if (this.worldRotationPoint == null) {
-          const worldCenter = BoundingBox.center(boundingBox);
-          this.worldRotationPoint =
-            depthBuffer != null
-              ? this.getWorldPoint(point, depthBuffer, worldCenter)
-              : camera.lookAt;
-        }
+    return this.transformCamera(({ camera, viewport, depthBuffer }) => {
+      if (this.worldRotationPoint == null) {
+        this.worldRotationPoint =
+          depthBuffer != null
+            ? this.getWorldPoint(point, depthBuffer, camera.lookAt)
```

Comment:
This is the main change. When the clicked point has no depth (as in the user clicked in the empty space), we now fall back to the camera's lookAt point instead of the worldCenter point.